python/example_code/lookoutvision/projects.py

```diff
@@ -0,0 +1,187 @@
+        :return project_arn: The ARN of the new project.
+        """
+
+        try:
+            # Create a project
+            logger.info("Creating project: %s", project_name)
+            response = lookoutvision_client.create_project(ProjectName=project_name)
+
+            logger.info("project ARN: %s", response["ProjectMetadata"]["ProjectArn"])
+
+            return response["ProjectMetadata"]["ProjectArn"]
+
+        except ClientError as err:
+            logger.exception("Couldn't create project: %s", err.response["Message"])
+            raise
+
+    @staticmethod
+    def delete_project(lookoutvision_client, project_name):
+        """
+        Deletes an Amazon Lookout for Vision Model
+        :param lookoutvision_client: An Amazon Lookout for Vision boto3 client.
+        :param: project_name - the name of the project that you want to delete.
+        """
+        try:
+
+            # Delete a project
+            logger.info("Deleting project: %s", project_name)
+            response = lookoutvision_client.delete_project(ProjectName=project_name)
+            logger.info("Deleted project ARN: %s ", response["ProjectArn"])
+
+        except ClientError as err:
+            logger.exception("Couldn't delete project: %s", err.response["Message"])
+            raise
+
+    @staticmethod
+    def list_projects(lookoutvision_client):
+        """
+        Lists information about the projects in your account.
+        :param lookoutvision_client: An Amazon Lookout for Vision boto3 client.
+        """
+
+        try:
+            response = lookoutvision_client.list_projects()
+
+            for project in response["Projects"]:
+
+                print("Project: " + project["ProjectName"])
+                print("\tARN: " + project["ProjectArn"])
+                print("\tCreated: " + str(["CreationTimestamp"]))
+
+                print("Datasets")
+                project_description = lookoutvision_client.describe_project(
+                    ProjectName=project["ProjectName"]
+                )
+                if len(project_description["ProjectDescription"]["Datasets"]) == 0:
+                    print("\tNo datasets")
+                else:
+                    for dataset in project_description["ProjectDescription"][
+                        "Datasets"
+                    ]:
+                        print("\ttype: " + dataset["DatasetType"])
+                        print("\tStatus: " + dataset["StatusMessage"])
+
+                print("Models")
+                # list models
+                response_models = lookoutvision_client.list_models(
+                    ProjectName=project["ProjectName"]
+                )
+                if len(response_models["Models"]) == 0:
+                    print("\tNo models")
+                else:
+                    for model in response_models["Models"]:
+                        print("\tVersion: " + model["ModelVersion"])
```

Comment:
Also, consider pulling this block into print_model function--somewhat hard to read with lots of lines with large indent.

Reply:
Fixed.